src/api/operator/types.ts

```diff
@@ -41,6 +42,7 @@ export type RawOrder = {
   kind: OrderKind
   partiallyFillable: boolean
   signature: string
+  status?: RawOrderStatusFromAPI
```

Comment:
Status will always be set

```suggestion
  status: RawOrderStatusFromAPI
```